warehouse/accounts/services.py

```diff
@@ -75,6 +76,35 @@ def check_password(self, userid, password):
 
         return True
 
+    def create_user(self, username, name, password, email,
+                    is_active=False, is_staff=False, is_superuser=False):
+        user = User(username=username,
+                    name=name,
+                    password=password,
+                    is_active=is_active,
+                    is_staff=is_staff,
+                    is_superuser=is_superuser)
+        user.last_login = datetime.datetime.now()
+        self.db.add(user)
+        self.db.flush()
```

Comment:
Do we need this flush here? I'm not super good at SQLAlchemy but my understanding was that it would generally flush whenever it needed to?